links.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="author" content="Tirthrajsinh">
+  <meta name="description" content="Useful links related to my faveroite club.">
+  <title>Links - Tirthrajisnh's Website</title>
+  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <!-- Found me! -->
+  
+</head>
+<body>
+  <div class="width-wrap">
+    <header>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="links.html">Links</a></li>
+          <li><a href="glossary.html">Glossary</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <h1>Useful Links</h1>
+      <p>Here are some links to websites that I find interesting or helpful:</p>
+      <ul>
+        <li><a href="https://www.w3schools.com" target="_blank">W3Schools - Learn Web Development</a></li>
+        <li><a href="https://developer.mozilla.org" target="_blank">MDN Web Docs - Web Documentation</a></li>
+        <li><a href="https://github.com" target="_blank">GitHub - Version Control and Collaboration</a></li>
+        <li><a href="https://www.youtube.com" target="_blank">YouTube - Tutorials and Inspiration</a></li>
+        <li><a href="https://unsplash.com" target="_blank">Unsplash - Free Images for Projects</a></li>
```

Comment:
Awesome that you've got additional links, but missing HTML Validator & Favicon Generator.
```suggestion
        <li><a href="https://unsplash.com" target="_blank">Unsplash - Free Images for Projects</a></li>
        <li><a href="https://validator.w3.org/" target="_blank" title="HTML Validator">HTML Validator</a></li>
        <li><a href="https://realfavicongenerator.net" target="_blank" title="Favicon Generator">Favicon Generator</a></li>
```